install/kubernetes/helm/istio/templates/rbac-beta.yaml

```diff
@@ -56,7 +56,7 @@ metadata:
 rules:
 - apiGroups: ["config.istio.io"] # istio CRD watcher
   resources: ["*"]
-  verbs: ["get", "list", "watch"]
+  verbs: ["create", "get", "list", "watch"]
```

Comment:
Why "create"? As far as I know CRDs are not created in the code, which is recommended practice.

Reply:
this is because of the race condition with CRDs, after CRDs are available to use, `create-config-job.yaml` triggers a job which install all the manifests that depend on the CRDs, this is only a container that runs `kubectl apply -f ...` so it needs permission to create resources from `config.istio.io`